### PR TITLE
TST: Replace Travis

### LINKED ISSUE #112: Ditch Travis CI as a recommended solution
Unless you want to pay them -- https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing

### PR DESCRIPTION
**Pull request by pllim**
Fix #112

### CODE REVIEW

File: guides/python-package.md [hunk 1]
```diff
@@ -60,15 +60,15 @@ Following [STScI policy on source code
...
+* Is it ITAR protected?  Then host it on internal git ITAR server.
 * Is it functional work?   Then it must be hosted on the STScI GitLab solution or the `spacetelescope` organization on GitHub
 * Will it have external collaborators?  Then it is recommended to host it on the `spacetelescope` organization on GitHub
 
 Code hosted on GitHub should follow the [STScI GitHub
 Policy](https://innerspace.stsci.edu/display/isec/GitHub).
 
 
-### Creating an new repository  
+### Creating an new repository
```
Comment (jhunkeler) [inline guides/python-package.md:71]: "Creating ~an~ a new repository"

File: guides/python-testing.md [hunk 2]
```diff
@@ -100,25 +100,18 @@ push events (e.g., commits to GitHub) or to run in a preset cadence
 
 For GitHub, here are some services available:
 
+* GitHub Actions: This is a paid service billed centrally to STScI.
+  Repositories on ``spacetelescope`` GitHub organization should use it,
+  if possible.
 * Jenkins CI: This is an in-house CI solution with virtually no limitation
   (but do not push your luck). It supports storing big data and artifacts in
   Artifactory. Its internal-only instance also supports direct access to
```
Comment (jhunkeler) [inline guides/python-testing.md:108]: The system does not allow for direct access to central storage.

File: guides/python-testing.md [hunk 3]
```diff
@@ -100,25 +100,18 @@ push events (e.g., commits to GitHub) or to run in a preset cadence
...
   Central Storage. Unfortunately, it is not easily accessible for non-STScI
   personnels. It currently only supports Linux operating system. For more
```
Comment (jhunkeler) [inline guides/python-testing.md:110]: MacOS is available but your mileage may vary due to an extreme lack of resources (i.e there's one box).

The machine can be accessed using the labels `macos`, or `osx`.

File: guides/python-testing.md (hunk 2 above)
Comment (pllim) [reply guides/python-testing.md:108]: I don't understand your suggested change here. Can you please clarify?

Comment (jhunkeler) [reply guides/python-testing.md:108]: The server doesn't allow you to access central storage at all.

Comment (pllim) [reply guides/python-testing.md:108]: I think the internal-only instance is referring to `JenkinsfileRT`.

Comment (jhunkeler) [reply guides/python-testing.md:108]: How about:

If STScI staff need to execute testing at regular intervals, or require direct access to central storage, please refer to the following documentation, [Users Guide: Running Regression Tests](https://innerspace.stsci.edu/display/SSR/Users+Guide%3A+Running+Regression+Tests) for more information.

Comment (pllim) [reply guides/python-testing.md]: Your text is applied. What about now?

**Pull request merged**